src/lib/components/editor/LinkBibleReferenceButton.svelte

```diff
@@ -344,6 +354,14 @@
                 </div>
             </div>
         {/if}
+        <label class="flex items-center">
+            <input type="checkbox" class="mr-2" bind:checked={updateAssociation} />
+            {#if existingReference}
+                <span>Update resource's Bible reference association</span>
```

Comment:
@jwinston-bn let me know what you think about this. It seemed like existing links needed a different message than newly created links.